### PR TITLE
Email notification for booking page

### PR DESCRIPTION
**Pull request by GyuoMi**
actually implementing something :NOWAYING:
4x

### CODE REVIEW

**Review by SahilDinanath — CHANGES_REQUESTED** [review]
appointments/+page.server.js


seems like the INSERT_INTO query got replaced with the old one

File: appointment-app/src/routes/appointments/+page.server.js [hunk 1]
```diff
@@ -63,7 +84,8 @@ export const actions = {
       eventStart.setHours(eventStart.getHours());
       eventEnd = new Date(eventListJson[i].end);
       eventEnd.setHours(eventEnd.getHours());
-      await database.mysqlconn.query('INSERT INTO BOOKING_TBL(CLIENT_ID,EVENT_ID,EVENT_START,EVENT_END,EVENT_TITLE) values(?,?,?,?,?)', [_clientID, eventId, eventStart.toISOString(), eventEnd.toISOString(), eventTitle]);
```
Comment (SahilDinanath) [inline appointment-app/src/routes/appointments/+page.server.js:66]: new insert statement is being replaced with old, will cause issue.

Comment (GyuoMi) [reply appointment-app/src/routes/appointments/+page.server.js:66]: fixed the issue

**Review by SahilDinanath — APPROVED** [review]
looking good

**Pull request merged**